Scoop/SearchRidesViewController.swift

```diff
@@ -0,0 +1,284 @@
+//
+//  SearchRidesViewController.swift
+//  Scoop
+//
+//  Created by Richie Sun on 4/7/23.
+//
+
+import GooglePlaces
+import UIKit
+
+class SearchRidesViewController: UIViewController {
+    
+    //MARK: Views
+    private let arrivalLabel = UILabel()
+    private let arrivalTextField = ImageTextField()
+    private let calendarIcon = UIImageView()
+    private let datePicker = UIDatePicker()
+    private let departureDateLabel = UILabel()
+    private let departureDateTextField = OnboardingTextField()
+    private let departureLabel = UILabel()
+    private let departureTextField = ImageTextField()
+    private let findTripsButton = UIButton()
+    private let stackView = UIStackView()
+    
+    // MARK: Data
+    private var arrivalPlace: GMSPlace?
+    private var departurePlace: GMSPlace?
+    private var tripDate: String = ""
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        tripDate = formatDate(dateToConvert: Date())
+        
+        setupHeader()
+        setupStackView()
+        setupLocationTextFields()
+        setupButton()
+        setupLabels()
+    }
+    
+    private func setupHeader() {
+        let dottedLineMultiplier = 0.52
+        let solidLineVerticalInset = -12.75
+        let solidLineMultiplier = 0.32
+        let screenSize = UIScreen.main.bounds
+        let dottedline = UIImageView(image: UIImage(named: "dottedline"))
+        
+        dottedline.contentMode = .scaleAspectFit
+        view.addSubview(dottedline)
+        dottedline.snp.makeConstraints { make in
+            make.height.equalTo(1)
+            make.width.equalTo(screenSize.width*dottedLineMultiplier)
+            make.trailing.equalToSuperview().inset(10)
+            make.top.equalTo(view.safeAreaLayoutGuide)
+        }
+        
+        let solidline = UIView()
+        solidline.backgroundColor = .black
+        view.addSubview(solidline)
+        solidline.snp.makeConstraints { make in
+            make.height.equalTo(1)
+            make.width.equalTo(screenSize.width*solidLineMultiplier)
+            make.trailing.equalToSuperview()
+            make.top.equalTo(view.safeAreaLayoutGuide).inset(solidLineVerticalInset)
```

Comment:
i can't tell if this indentation looks weird or if my github is just bugging out